base_multi_company/hooks.py

```diff
@@ -57,12 +57,17 @@ def post_init_hook(cr, rule_ref, model_name):
         table_name = model._fields['company_ids'].relation
         column1 = model._fields['company_ids'].column1
         column2 = model._fields['company_ids'].column2
-        SQL = """
+        sql = """
             INSERT INTO %s
             (%s, %s)
-            SELECT id, company_id FROM %s WHERE company_id IS NOT NULL
-        """ % (table_name, column1, column2, model._table)
-        env.cr.execute(SQL)
+            SELECT id, company_id FROM %s AS tb1
```

Comment:
Why is this needed?

Reply:
That is because when uninstall the module partner_multi_company and after install the same module partner_multi_company this crash and is not is possible install.